Runtime/BugSplat.cs

```diff
@@ -206,6 +207,36 @@ string version
 #endif
         }
 
+        /// <summary>
+        /// Constructs and returns a BugSplat object from BugSplatOptions
+        /// </summary>
+        /// <param name="options">collection of options which can be used to configure a BugSplat object </param>
+        /// <param name="application">Your application's name (must match value used to upload symbols)</param>
+        /// <param name="version">Your application's version (must match value used to upload symbols)</param>
+        public static BugSplat CreateFromOptions(BugSplatOptions options, string application, string version)
```

Comment:
This constructor feels a little weird, could we make `application` and` version` optional properties on `BugSplatOptions` that are initialized to `Application.productName` and `Application.version`?

Reply:
I've updated BugSplatOptions to contain fields for these values under the `Optional` category. In `CreateFromOptions`, if the value passed in is null it defaults to the UnityEngine Application version/app. Additionally, the tooltips have been created to reflect this in the `BugSplatOptions`.